README.md

```diff
@@ -148,7 +150,7 @@ To use this tool on a Pantheon site that does not have multidev environments sup
 
 ## Adding More Providers
 
-At the moment, the build:project:create command only supports GitHub and Circle CI. It is possible to add other providers.
+At the moment, the build:project:create command only supports GitHub, BitBucket, GitLab, and Circle CI. It is possible to add other providers.
```

Comment:
We should be more specific in this list and clearly define which git hosts and CI services are supported together.

| git host  | CI service |
|:---------:|:----------:|
|   GitHub  |  CircleCI  |
|   GitLab  |   GitLab   |
| BitBucket |  CircleCI  |